source/includes/details-for-a-single-app.md

```diff
@@ -186,29 +186,29 @@ Field                    | Description
 **all_histogram**        | The histogram breakdown of ratings for all versions of this app. A dictionary with the keys 1-5 (as strings) mapped onto number of reviews with that number of stars.
 **all_rating**           | The average rating for all versions of this app.
 **all_rating_count**     | The number of ratings for all versions of this app.
-**all_reviews**          | The number of reviews for all versions of this app, if available as a summarized number.
+**all_reviews**          | The number of reviews for all versions of this app, if available as a summarized number. ![itunes_only](../images/itunes_logo.jpg)
```

Comment:
Can you check how this would look if we'd have another column before description called "Availability" or "Platform" or
"Only On", "Restricted to", something like that? And then icons for the fields where it's available only on one.

Reply:
I tried that since yesterday, it'd look much better...except that I'm having this style/alignment issue, when `json-inline` comes into action and breaks the table...atill trying to figure what would be proper fix for it:

![logo_align](https://user-images.githubusercontent.com/3522071/47020433-096def80-d162-11e8-9cc1-1c718287246e.png)